src/app/pages/questions/components/question/question.component.html

```diff
@@ -69,6 +69,7 @@
           *ngSwitchCase="'info'"
           [sections]="question.select_choices_or_calculations"
           [hasFieldLabel]="question.field_label.length > 0"
+          [image]="question.field_annotation?.image"
```

Comment:
is this for specifying the path of the image via field annotation in redcap?
I would suggest rather using the protocol for this

Reply:
Sort of, like here: https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/blob/master/questionnaires/romberg_test/romberg_test_armt.json I think this was just manually added?